src/removeUnusedPages.js

```diff
@@ -0,0 +1,85 @@
+const fs = require("fs");
+const path = require("path");
+
+/** @typedef {'nodejs' | 'dotnet' | 'java' | 'python'} LanguageFolder */
+
+/** @type {Record<LanguageFolder, string[]} */
+const languageExceptions = {
```

Comment:
It is a little tricky to differentiate languageExceptions from exceptions. What does that mean?

Reply:
first is for some languages only and the other one for all the languages.